llvm/include/llvm/Frontend/HLSL/HLSLResource.h

```diff
@@ -13,6 +13,7 @@
 #ifndef LLVM_FRONTEND_HLSL_HLSLRESOURCE_H
 #define LLVM_FRONTEND_HLSL_HLSLRESOURCE_H
 
+#include "llvm/ADT/StringRef.h"
```

Comment:
StringRef is used in line 39:
    StringRef getSourceType();

Is it OK to remove the include?

Reply:
This adds the StringRef include so that that continues working...